ui/components/MesheryMeshInterface/PatternServiceForm.js

```diff
@@ -19,6 +20,21 @@ const useStyles = makeStyles(() => ({
   },
   formWrapper : {
     width : "100%"
+  },
+  settingsIcon : {
+    color : "black"
+  },
+  appTabs : {
+    width : 128,
+    overflow : 'hidden',
+    transition : 'width 0.5s',
+    '&.Mui-disabled' : {
```

Comment:
Should it have space before className???
```suggestion
    '& .Mui-disabled' : {
```

Reply:
Not firm though, check if that works in your system.

Reply:
@Abhishek-kumar09 both of the approaches are correct with or without space as per MUI for eg:
![Screenshot from 2022-01-05 19-06-35](https://user-images.githubusercontent.com/65453826/148226562-ea3df2f9-5959-4622-9b1c-829c803e9979.png)

Reply:
Oh Thanks for letting me know about it. Nice.